shell/common/fixtures/shell_test.dart

```diff
@@ -163,6 +163,17 @@ void canDecompressImageFromAsset() {
 
 List<int> getFixtureImage() native 'GetFixtureImage';
 
+@pragma('vm:entry-point')
+void canRegisterImageDecoders() {
+  decodeImageFromList(
+    // The test ImageGenerator will always behave the same regardless of input.
+    Uint8List.fromList([0]),
```

Comment:
nit: we normally type specify arrays, but this could also just be `Uint8List(1)`, since Dart zero initializes typed data anyway.

Reply:
Neat, done.